services/front/src/main.tsx

```diff
@@ -17,6 +17,36 @@ import { MainLoading } from "./MainLoading";
 import { Buffer } from "buffer";
 globalThis.Buffer = Buffer;
 
+navigator.serviceWorker
+ .register(
+   process.env.NODE_ENV === 'production' ? '/sw.js' : '/dev-sw.js?dev-sw'
```

Comment:
where is dev-sw.js?

Reply:
why do we need a separate file for dev?

Reply:
https://vite-pwa-org.netlify.app/guide/development.html#injectmanifest-strategy

Reply:
Everything is explained in this link

Reply:
According to the documentation we must use:
```ts
{
      devOptions: {
        enabled: true,
        type: "module",
      }
}
```
as well

Reply:
have you tested in dev mode?

Reply:
OK I  merge we will see later.